automations/python/shared/data.py

```diff
@@ -12,16 +12,20 @@
 
 
 @cache
-def get_data(file_name: str) -> dict:
+def get_data(file_name: str, encoding: str = None) -> dict:
```

Comment:
Was this necessary for reading the file? Python _should_ [default to `utf-8` when opening files](https://docs.python.org/3/howto/unicode.html#encodings) if it's not set explicitly. Even if we needed to make this explicit here, we should just default to `utf-8` anyway and avoid the need for the `if encoding` block below.

Reply:
It seems that [the encoding used by open() is platform-specific](https://docs.python.org/3/library/functions.html#open), although utf-8 seems to be the most prevelant. On my platform running open() on labels.yml results in `UnicodeDecodeError: 'charmap' codec can't decode byte 0x8f in position 980: character maps to <undefined>`. Specifying "utf-8" as encoding method fixes this. 
I agree on making it the default and I removed the if statement, get_data() now defaults to utf-8.

Reply:
What platform are you using @Tomvth? Since these actions will be running inside GitHub Actions, the default encoding will be UTF-8.